### ISSUE TITLE
Error: index 30 is out of bounds for axis 0 with size 30

### ISSUE THREAD

**Issue by amper43** [hastic/hastic-server#413]
### Error
```
index 30 is out of bounds for axis 0 with size 30
  File "/home/jo/extra/hastic-server/analytics/analytics/utils/common.py", line 95, in get_av_model
    av_val.append(val[i])
  File "/home/jo/extra/hastic-server/analytics/analytics/models/jump_model.py", line 48, in do_fit
    self.state['pattern_model'] = utils.get_av_model(learning_info['patterns_list'])
  File "/home/jo/extra/hastic-server/analytics/analytics/models/model.py", line 87, in fit
    self.do_fit(dataframe, labeled, deleted, learning_info)
  File "/home/jo/extra/hastic-server/analytics/analytics/detectors/pattern_detector.py", line 46, in train
    new_cache = self.model.fit(dataframe, segments, cache)
  File "/usr/lib/python3.6/concurrent/futures/thread.py", line 56, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/lib/python3.6/concurrent/futures/thread.py", line 69, in _worker
    work_item.run()
  File "/usr/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
```
### Model
Jump

### Method
get_av_model

### Args and locals
* patterns_list:
```
[
array([0.05609999969601631, 0.05010000243782997, 0.04970000311732292,
       0.018699999898672104, 0.0340999998152256, 0.08010000362992287,
       0.04650000110268593, 0.019700001925230026, 0.020299997180700302,
       0.017400000244379044, 0.03960000351071358, 0.045899998396635056,
       0.08010000362992287, 0.034300003200769424, 0.04350000247359276,
       0.01419999822974205, 0.019499998539686203, 0.012200001627206802,
       0.045899998396635056, 0.07279999926686287, 0.0, 0.0430000014603138,
       0.0340999998152256, 0.025199998170137405, 0.12289999797940254,
       0.019099999219179153, 0.013099998235702515, 0.06389999762177467,
       0.010400000959634781, 0.05940000340342522, 0.025299999862909317,
       0.08030000701546669, 0.014899998903274536, 0.03579999879002571,
       0.02710000053048134, 0.02069999650120735, 0.03070000186562538,
       0.029900003224611282, 0.017299998551607132, 0.03829999640583992,
       0.02770000323653221, 0.02240000292658806, 0.011500000953674316,
       0.00910000130534172, 0.013000000268220901, 0.06439999863505363,
       0.04769999906420708, 0.04340000078082085, 0.0633000023663044,
       0.044499997049570084, 0.011500000953674316, 0.03960000351071358,
       0.02590000256896019], dtype=object),
array([0.06119999662041664, 0.04009999707341194, 0.01759999990463257,
       0.0819999985396862, 0.034700002521276474, 0.06339999660849571,
       0.041399996727705, 0.03169999644160271, 0.010600000619888306,
       0.04399999603629112, 0.029099997133016586, 0.017400000244379044,
       0.04740000143647194, 0.04650000110268593, 0.017799999564886093,
       0.0, 0.03169999644160271, 0.044099997729063034,
       0.039799999445676804, 0.04520000144839287, 0.13079999759793282,
       0.09099999442696571, 0.03259999677538872, 0.05899999663233757,
       0.09310000017285347, 0.03869999572634697, 0.1103999949991703,
       0.07690000161528587, 0.06240000203251839, 0.07070000097155571,
       0.06890000030398369, 0.029599998146295547, 0.03449999913573265,
       0.0672999955713749, 0.0575999952852726, 0.07279999926686287,
       0.05389999970793724, 0.061799999326467514, 0.07189999893307686,
       0.02459999918937683, 0.03309999778866768, 0.05490000173449516,
       0.09559999778866768, 0.08149999752640724, 0.061799999326467514,
       0.09060000255703926, 0.06259999796748161, 0.048700001090765,
       0.05140000209212303, 0.06119999662041664, 0.013999998569488525,
       0.08289999887347221, 0.06689999625086784], dtype=object),
array([0.03880000114440918, 0.02459999918937683, 0.03800000250339508,
       0.014299999922513962, 0.08389999717473984, 0.049000002443790436,
       0.049800001084804535, 0.04429999738931656, 0.04050000011920929,
       0.05300000309944153, 0.0429999977350235, 0.01900000125169754,
       0.01680000126361847, 0.037699997425079346, 0.028700001537799835,
       0.013300001621246338, 0.05650000274181366, 0.04119999706745148,
       0.035999998450279236, 0.011300001293420792, 0.012200001627206802,
       0.029899999499320984, 0.03519999980926514, 0.014400001615285873,
       0.022199999541044235, 0.0, 0.028999999165534973,
       0.020800001919269562, 0.01599999889731407, 0.02019999921321869,
       0.016699999570846558, 0.050599999725818634, 0.05210000276565552,
       0.06440000236034393, 0.030299998819828033, 0.028300002217292786,
       0.026100002229213715, 0.01080000028014183, 0.012800000607967377,
       0.04410000145435333, 0.04439999908208847, 0.026799999177455902,
       0.0065000019967556, 0.012600000947713852, 0.034699998795986176,
       0.04330000281333923, 0.02149999886751175, 0.0210999995470047,
       0.07649999856948853, 0.04020000249147415, 0.008700001984834671,
       0.037299998104572296, 0.045100003480911255], dtype=object),
array([0.05230000242590904, 0.10490000620484352, 0.08080000057816505,
       0.05389999970793724, 0.03590000048279762, 0.02540000155568123,
       0.07979999855160713, 0.02980000153183937, 0.07070000097155571,
       0.059700001031160355, 0.06450000032782555, 0.06399999931454659,
       0.04480000212788582, 0.05480000004172325, 0.04650000110268593,
       0.06950000301003456, 0.014000002294778824, 0.026099998503923416,
       0.06730000302195549, 0.011900000274181366, 0.07910000160336494,
       0.060600001364946365, 0.028299998492002487, 0.04780000075697899,
       0.024900000542402267, 0.024900000542402267, 0.03150000050663948,
       0.04859999939799309, 0.0430000014603138, 0.030800003558397293,
       0.06120000407099724, 0.03839999809861183, 0.07579999789595604,
       0.08949999883770943, 0.02770000323653221, 0.0, 0.05620000138878822,
       0.016300000250339508, 0.0844000019133091, 0.016100000590085983,
       0.04430000111460686, 0.07030000165104866, 0.06690000370144844,
       0.01600000262260437, 0.038600001484155655, 0.027399998158216476,
       0.04019999876618385, 0.050700005143880844, 0.05350000038743019,
       0.012300003319978714, 0.009200002998113632, 0.04920000210404396,
       0.04230000451207161], dtype=object),
array([0.04190000146627426, 0.0130000039935112, 0.04240000247955322,
       0.028400003910064697, 0.027000002562999725, 0.007100000977516174,
       0.03660000115633011, 0.0689999982714653, 0.09300000220537186,
       0.052400000393390656, 0.04330000281333923, 0.05120000243186951,
       0.016599997878074646, 0.04360000044107437, 0.056699998676776886,
       0.07380000501871109, 0.055799998342990875, 0.0,
       0.016699999570846558, 0.05210000276565552, 0.031000003218650818,
       0.030400000512599945, 0.008699998259544373, 0.04829999804496765,
       0.11219999939203262, 0.03680000454187393, 0.023600004613399506,
       0.045100003480911255, 0.039000004529953, 0.02229999750852585],
      dtype=object)
]
```
* i: `30`
* x: `53`
* av_val: `[]`
* model_pat:
```
[0.04190000146627426, 0.0130000039935112, 0.04240000247955322, 0.028400003910064697, 0.027000002562999725, 0.007100000977516174, 0.03660000115633011, 0.0689999982714653, 0.09300000220537186, 0.052400000393390656, 0.04330000281333923, 0.05120000243186951, 0.016599997878074646, 0.04360000044107437, 0.056699998676776886, 0.07380000501871109, 0.055799998342990875, 0.0, 0.016699999570846558, 0.05210000276565552, 0.031000003218650818, 0.030400000512599945, 0.008699998259544373, 0.04829999804496765, 0.11219999939203262, 0.03680000454187393, 0.023600004613399506, 0.045100003480911255, 0.039000004529953, 0.02229999750852585]
```

**Comment by jonyrock** [comment]
@rozetko @amper43  25 days ago? seriously? what is the status of this issue?

**Comment by rozetko** [comment]
Fixed via https://github.com/hastic/hastic-server/pull/427
